packages/flame/lib/src/game/camera/camera_wrapper.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:ui';
+import '../../../components.dart';
+import 'camera.dart';
+
+/// Do not use.
```

Comment:
I don't think we need this comment, since we are still on RCs everything can be changed at any time